less/navigation.less

```diff
@@ -0,0 +1,36 @@
+
+.container {
+    nav{
+        width: 100%;
+        height: 105px;
+        display: flex;
+        justify-content: center;
+        flex-wrap: wrap;
+        background-color: @navigation-background;
+        h1{
+            width:100%;
+            margin-top: 15px;
+            font-size: 4rem;
+            color: @navigation-font-color;
+            .center;
+        }
+        .navigation {
+            width: 67%;
+            .center;
+            @media (max-width: 500px){
```

Comment:
Perfect nesting, and use of nested at rules. Don't forget about the escape character, and set a variable for your media query so you can use `@media @phone`. The biggest benefit of that is that you can change that break point in just one place instead of many, which is a big deal in larger projects.